src/models/LayoutConfig.ts

```diff
@@ -112,6 +112,19 @@ export class LayoutConfig {
         };
     };
 
+    public static UpgradeLayout = (layout: { layoutVersion: 1 | 2, docked: any, floating: any }) => {
+        // Upgrade to V2 if required
+        if (layout.layoutVersion === 1) {
+            const spatialProfileWidgets = findDeep(layout, item => item.id === "spatial-profiler");
+            for (const widget of spatialProfileWidgets) {
+                if (widget.widgetSettings?.coord) {
```

Comment:
coord is not nested inside widgetSettings in ver1

Reply:
oops, ok. Will fix now :)